key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
What is this used for?

I can see the keys used in the tests that are fixtures - but I dislike this being in the base dir for the app.

In this directory it has no context associated with it. Why is it there? What is it used for? Who knows.

Can we remove if it is not being used - or find a better place for it where the directory that it is in gives it some context so people who look at the app can understand its purpose.

Reply:
This file is a mistake - didn't notice it, removed.